docs/source/app_developers_guide/intro_xo_transaction_family.rst

```diff
@@ -2,227 +2,227 @@
 Introduction to the XO Transaction Family
 *****************************************
 
-What is XO?
-===========
-
-*XO* is an example transaction family contained within the Sawtooth SDK. It is an
-implementation of the popular game *Tic-tac-toe* (otherwise known as
-*Noughts and Crosses* or *X's and O's*).
+XO is an example transaction family that implements the game
+`tic-tac-toe <https://en.wikipedia.org/wiki/Tic-tac-toe>`_,
+also known as *Noughts and Crosses* or *X's and O's*.
+We chose XO as an example transaction family for Sawtooth because of its
+simplicity, global player base, and straightforward implementation as a computer
+program. This transaction family demonstrates the functionality of Sawtooth;
+in addition, the code that implements it serves as a reference for building
+other transaction processors.
+
+This section introduces the concepts of a Sawtooth transaction family with XO,
+summarizes XO game rules, and describes how use the ``xo`` client application
+to play a game of tic-tac-toe on the blockchain.
+
+
+About the XO Transaction Family
+===============================
+
+The XO transaction family defines the data model and business logic for
+playing tic-tac-toe on the blockchain by submitting transactions for `create`,
+`take`, and `delete` actions. For more information, see
+:doc:`/transaction_family_specifications/xo_transaction_family`
 
-*X's and O's*-style games have been played across the world for many centuries. The
-origin of this type of game is unclear; however, some historians believe that the first
-version of *Tic-tac-toe* may have originated in ancient Egypt. Others believe that the
-modern *Tic-tac-toe* is an evolution of a game known as *Terni Lapilli*, which was
-popular within the Roman Empire.
+The XO transaction family includes:
 
-*Tic-tac-toe* also has historical significance within the field of Computer Science. In
-1952, it became the basis of one of the first-ever video games when
-`OXO <https://en.wikipedia.org/wiki/OXO>`_ was developed by Alexander S. Douglas at the
-University of Cambridge. More information regarding the origins and properties of
-*Tic-tac-toe* can be found `here <https://en.wikipedia.org/wiki/Tic-tac-toe>`_.
+ * Transaction processors in several languages, including Go (``xo-tp-go``),
+   JavaScript (``xo-tp-js``), and Python (``xo-tp-python``). These transaction
+   processors implement the business logic of XO game play.
 
-*XO* was chosen as an example transaction family for Sawtooth due to its simplicity,
-global player base, and straightforward implementation as a computer program. This example
-transaction family demonstrates the functionality of Sawtooth, and the code that implements
-it serves as a reference for building other transaction processors.
+ * An ``xo`` client: A set of commands that provide a command-line interface
+   for playing XO. The ``xo`` client handles the constructing and submission
```

Comment:
should we use "construction and submission" instead of "constructing and submission"